scipy/signal/tests/test_waveforms.py

```diff
@@ -72,6 +72,28 @@ def test_linear_freq_02(self):
         abserr = np.max(np.abs(f - chirp_linear(tf, f0, f1, t1)))
         assert_(abserr < 1e-6)
 
+    def test_linear_complex_power(self):
+        method = 'linear'
+        f0 = 1.0
+        f1 = 2.0
+        t1 = 1.0
+        t = np.linspace(0, t1, 100)
+        w_real = waveforms.chirp(t, f0, t1, f1, method, complex=False)
+        w_complex = waveforms.chirp(t, f0, t1, f1, method, complex=True)
+        w_pwr_r = np.var(w_real)
+        w_pwr_c = np.var(w_complex)
+        
+        # Making sure that power of the real part is not affected with 
+        # complex conversion operation
+        err = w_pwr_r - np.real(w_pwr_c)
+
+        assert_(err < 1e-6)
```

Comment:
new tests should use plain asserts
```suggestion
        assert err < 1e-6
```